packages/frameworks/src/frameworks.ts

```diff
@@ -6,7 +6,7 @@ import { readConfigFile } from './read-config-file';
 
```

Comment:
We're gonna want to do the changes in this file in a separate PR, since we'll want the `@vercel/gatsby` package to be already published + public on npm before we make this change.